File: java/SpringMvc/ServletMvc/src/main/java/org/zerock/servletmvc/todo/dto/TodoDTO.java
package org.zerock.servletmvc.todo.dto;

import java.time.LocalDate;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TodoDTO {

  private Long tno;
  private String title;
  private LocalDate dueDate;
  private boolean finished;

}

File: Kotlin/KT_todo/src/main/kotlin/org/ruu/kt/todo/todos/dto/TodoDTO.kt
package org.ruu.kt.todo.todos.dto

import org.ruu.kt.todo.todos.enum.Priority
import org.ruu.kt.todo.todos.enum.TodoStatus

data class TodoDTO(
    val id: Long? = null,
    val title: String,
    val content: String? = null,
    val todoStatus: TodoStatus,
    val priority: Priority = Priority.MEDIUM
)